src/EPR.Calculator.Frontend/Controllers/CalculationRunNameController.cs

```diff
@@ -14,16 +15,20 @@ public IActionResult Index()
         }
 
         [HttpPost]
-        public IActionResult RunCalculator(string calculationName)
+        public IActionResult RunCalculator(InitiateCalculatorRunModel calculationRunName)
         {
-            if (string.IsNullOrWhiteSpace(calculationName))
+            if (!this.ModelState.IsValid)
             {
-                this.ViewBag.Errors = CreateErrorViewModel();
+                var errorMessages = this.ModelState.Values.SelectMany(x => x.Errors).Select(e => e.ErrorMessage);
+                this.ViewBag.Errors = CreateErrorViewModel(errorMessages.First());
```

Comment:
You are only showing first message. Should you change it display more than one error message

Reply:
Why do we need show more than one message here? As per my validation I have to show separately for each. We can have quick call to discuss on this.

Reply:
_Uday Denduluri (ext) I had a call with Uma, We need to validate on not more than 100 characters first. If it's more than 100 and having some special characters in it then it we shouldn't show 2 error messages at a time (that's what mention in User Story). We only show More than 100 error message. Once user fixed and it's under 100 then will check Alpha Numeric. Hope this Answer your question._

Got this confirmed that we only showing one error message at a time. Thanks